package.json

```
@@ -6,7 +6,7 @@
   "dependencies": {
     "async": "^2.6.3",
     "mongodb": "^2.1.2 || ^3.0.0 || ^4.0.0",
-    "sharedb": "^1.0.0-beta"
+    "sharedb": "^1.9.1-beta"
```

We're out of beta now

```suggestion
    "sharedb": "^1.9.1"
```